### PR TITLE
MF-1424 - Enhance documentation for Writer

### LINKED ISSUE #1422: Fix documentation for Writers
In the documentation on the page

https://mainflux.readthedocs.io/en/latest/storage/

is written:

"...For the sake of simplicity, nested JSON objects are flatten to a single JSON object, using composite keys with the default separator /. This implies that the separator character (/) is not allowed in the JSON object key...."

The separator is hardcoded as a const in the code and therefore can not be changed

https://github.com/mainflux/mainflux/blob/master/pkg/transformers/json/transformer.go
Line 15: const sep = "/"

 This is a little bit misleading. Please fix the documentation.

### LINKED ISSUE #1424: Enhance documentation for Writers
This sounds very obvious, but I think it is important for the sake of completeness.

In the documentation for the writers on the page

https://mainflux.readthedocs.io/en/latest/storage/

the possible values for MF_MONGO_WRITER_TRANSFORMER are not pointed out. 

There is only written that the default is SenML, but the possibility for using JSON is not mentioned.

### PR DESCRIPTION
**Pull request by dborovcanin**
### What does this do?
This pull request enhances the Storage section of the docs.

### Which issue(s) does this PR fix/relate to?
This pull request resolves [#1424](https://github.com/mainflux/mainflux/issues/1424) and closes [#1422](https://github.com/mainflux/mainflux/issues/1422).

### List any changes that modify/break current functionality
There are no such changes.

### Have you included tests for your changes?
No.

### Did you document any new/modified functionality?
Yes.

### CODE REVIEW

File: docs/storage.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ In order to run these services, core services, as well as the network from the c
 
 ## Writers
 
-Writers provide an implementation of various `message writers`. Message writers are services that consume Mainflux messages, transform them to desired format and store them in specific data store. There are two types of transformers: JSON and SenML. transformer type is set using the following environment variables: `MF_CASSANDRA_WRITER_TRANSFORMER`, `MF_POSTGRES_WRITER_TRANSFORMER`, `MF_INFLUX_WRITER_TRANSFORMER` and `MF_MONGO_WRITER_TRANSFORMER`. The default value is `SenML` Transformer.
+Writers provide an implementation of various `message writers`. Message writers are services that consume Mainflux messages, transform them to desired format and store them in specific data store. There are two types of transformers: `JSON` and `SenML`. transformer type is set using the following environment variables: `MF_CASSANDRA_WRITER_TRANSFORMER`, `MF_POSTGRES_WRITER_TRANSFORMER`, `MF_INFLUX_WRITER_TRANSFORMER` and `MF_MONGO_WRITER_TRANSFORMER`. Valid options for this env var are `JSON` and `SenML` and those values are not case sensitive. The default Transformer is `SenML`.
```
Comment (manuio) [inline docs/storage.md]: I'd use `environment variable`

**Review by nmarcetic — DISMISSED** [review]

**Review by mteodor — DISMISSED** [review]

**Comment by drasko** [comment]
DCO is failing

**Review by manuio — DISMISSED** [review]
LGTM

**Comment by nmarcetic** [comment]
LGTM

@dusanb94 PLease resolve conflicts.

**Review by nmarcetic — DISMISSED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**